### DISCUSSION TITLE
Uploading proof of academic status didn't get processed

### DISCUSSION THREAD

**Discussion by neuroben** [community/community#106936] [GitHub Education]
### Hi everyone,

I have a student ID card, which has two sides. One side shows the expiration date, and the other side displays my name and the name of my school. The problem is, no matter how I try to upload the edited photo of both sides, I keep getting rejected. I can only take a photo of one side through the webcam, so I'm not sure what to do. I would appreciate it if someone could help me with this because my free trial of Copilot will expire soon. Thank you in advance.

**Comment by neuroben** [comment]
Any suggestions what should I do?

**Comment by faridnec** [comment]
Hey, I also had a similar issue in which my ID displays my name at the front of its page and does not display the validity date. I uploaded it only to the front page and they accepted it. I did not upload through the file tho, I took the photo of it directly with the camera provided there and it solved the issue as well as they accepted my application.

**Reply by neuroben** [reply]
thanks i'll try
